src/vast_tracker.js

```diff
@@ -571,6 +641,9 @@ export class VASTTracker extends EventEmitter {
    *
    */
   track(eventName, { macros = {}, once = false } = {}) {
+    if (typeof eventName !== 'string' || typeof macros !== 'object') {
```

Comment:
Not sure if the test is usefull as we already do some checks on `eventName` below